code/modules/credits/episode_name.dm

```diff
@@ -106,8 +106,12 @@
 		episode_names += new /datum/episode_name("WHAT HAPPENS WHEN YOU MIX MOMMIS AND COMMERCIAL-GRADE PACKING FOAM", "There was a powergrid with [score["maxpower"]]W, and 1 or more MoMMIs playing.", 250)
 	if(score["singuloosed"] > 0)
 		episode_names += new /datum/episode_name("[pick("THE SINGULARITY GETS LOOSE [prob(50) ? "(AGAIN)" : ""]", "THE CREW'S ENGINE SUCKS", "CONTAINMENT FAILURE", "THE GOOSE IS LOOSE")]", "There Singularity got loose this round.", 300) //no "singularity's day out" please we already have enough
+	if(score["kudzugrowth"] > 200)
+		episode_names += new /datum/episode_name("[pick("REAP WHAT YOU SOW", "FARM ILL", "SEEDY BUSINESS", "[uppr_name] AND THE BEANSTALK", "IN THE GARDEN OF EDEN")]", "[score["kudzugrowth"]] tiles worth of Kudzu were grown in total this round.", min(1500, score["kudzugrowth"]))
+	if(score["oremined"] > 500)
```

Comment:
The irresistible rise thing was a bug, I used max instead of min, hasn't fired once since I fixed it

You are totally right about nobody ever seeing ore mined but those are good puns so I refuse to not leave it in as something very rare